data/langdb.yaml

```diff
@@ -488,6 +488,7 @@ languages:
   twd: [Latn, [EU], Tweants]
   ty: [Latn, [PA], reo tahiti]
   tyv: [Cyrl, [AS], тыва дыл]
+  tzl: [Latn, [WW], Talossan]
```

Comment:
I mapped it as an worldwide language, because other constructed languages (like Esperanto and Lojban) were also mapped as such.
Also, there are many people outside of Talossa who speak/understand the language.